### PR TITLE
Documentation improvements for the validator migration

### PR DESCRIPTION
**Pull request by jsdanielh**
Documentation improvements for the validator registration guide.

### CODE REVIEW

File: migration/migration-validators.md [hunk 1]
```diff
@@ -45,13 +49,13 @@ You will need funds to pay for the validator registration transactions and for t
...
+To register your validator and the corresponding keys on the PoW Mainnet, you must specify the arguments as follows:
 
 ```shell
 node validator-registration.js --validator validator-keys.json --network main
 ```
 
-The tool will connect to the Nimiq PoW Mainnet chain. Upon consensus, the tool creates and signs the validator registration transactions and sends them to the network, effectively registering your validator. Note that these transactions are a bundle of 6, and each one has a nominal fee of 1 Luna. Please ensure you have funded your validator address &mdash; as instructed in the previous step &mdash; as the transaction fees are paid from this address.
+The tool will connect to the Nimiq PoW Mainnet chain. Upon consensus, the tool creates and signs the validator registration transactions and send them to the network, effectively registering your validator. Note that these transactions are a bundle of 6, and each one has a nominal fee of 1 Luna. Please ensure you have funded your validator address &mdash; as instructed in the previous step &mdash; as the transaction fees are paid from this address.
```
Comment (sisou) [inline migration/migration-validators.md]: `sends` was right, as it relates to the tool.

File: migration/migration-validators.md [hunk 2]
```diff
@@ -95,18 +103,22 @@ The Activation tool establishes a connection with the PoW chain via RPC, extract
...
 ```shell
 node clients/nodejs/index.js --dumb --network=main --rpc=8648 --wallet-seed=private_keypublic_key
 ```
 
+Note that `private_keypublic_key` is the concatenation of your private key and public key in that order.
+
 Before executing the activation tool, make sure you run ```cargo build --release``` within the PoS repository to compile the code.
 
-Once you are in consensus in the PoW chain, proceed to execute the migration tool by running the following command **in the PoS chain**, including the path to the configuration file containing your validator data and specifying the RPC server to be used:
+Once you are in consensus in the PoW chain, proceed to execute the migration tool by running the following command **in the PoS chain client**, including the path to the configuration file containing your validator data and specifying the RPC server to be used:
```
Comment (sisou) [inline migration/migration-validators.md]:
```suggestion
Once you are in consensus in the PoW chain, proceed to execute the migration tool by running the following command **in the PoS chain client** directory, including the path to the configuration file containing your validator data and specifying the RPC server to be used:
```

File: migration/migration-validators.md [hunk 3]
```diff
@@ -95,18 +103,22 @@ The Activation tool establishes a connection with the PoW chain via RPC, extract
...
 
 ```shell
 cargo run --release --bin nimiq-pow-migration --url "url-according-your-configuration" --config client.toml
 ```
 
+Where `url-according-your-configuration` is the PoW client RPC url. For instance: `http://172.0.0.1:8648`.
```
Comment (sisou) [inline migration/migration-validators.md]:
```suggestion
Where `url-according-your-configuration` is the PoW client RPC url. For example: `http://172.0.0.1:8648`.
```

File: migration/migration-validators.md [hunk 4]
```diff
@@ -9,7 +9,11 @@ The Validator Registration tool facilitates the pre-registration process for val
 - Keys and corresponding address generation
 - Validator registration via transactions
 
- First, you need to clone the [Nimiq Validator Registration Tool repository](https://github.com/nimiq/validator-registration-tool) and install its dependencies by executing `yarn` inside the repository.
+### Prerequisites
+
+1. Make sure you have [Node.js](https://nodejs.org) >= v16.20.2 installed.
```
Comment (Eligioo) [inline migration/migration-validators.md]: Node V18 is the latest maintained version. We shouldn't endorse any unmaintained versions: https://nodejs.org/en/about/previous-releases

File: migration/migration-validators.md [hunk 5]
```diff
@@ -95,18 +103,22 @@ The Activation tool establishes a connection with the PoW chain via RPC, extract
...
 ```shell
 node clients/nodejs/index.js --dumb --network=main --rpc=8648 --wallet-seed=private_keypublic_key
 ```
 
+Note that `private_keypublic_key` is the concatenation of your private key and public key in that order.
+
 Before executing the activation tool, make sure you run ```cargo build --release``` within the PoS repository to compile the code.
 
-Once you are in consensus in the PoW chain, proceed to execute the migration tool by running the following command **in the PoS chain**, including the path to the configuration file containing your validator data and specifying the RPC server to be used:
+Once you are in consensus in the PoW chain, proceed to execute the migration tool by running the following command **in the PoS chain client** directory, including the path to the configuration file containing your validator data and specifying the RPC server to be used:
```
Comment (Eligioo) [inline migration/migration-validators.md]:
```suggestion
Once you have consensus on the PoW chain, proceed to execute the migration tool by running the following command **in the PoS chain client** directory, including the path to the configuration file containing your validator data and specifying the RPC server to be used:
```

**Comment by onmax** [comment]
Deployed in prod https://www.nimiq.com/developers/migration/

**Comment by Eligioo** [comment]
> Deployed in prod https://www.nimiq.com/developers/migration/

@onmax Gives me a 404 on mobile

**Pull request merged**